### ISSUE TITLE
less variables for ships

### ISSUE THREAD

**Issue by icydee** [plainblack/Lacuna-Server-Open#443]
In order to group ships into fleets they need to have exactly the same statistics.

Currently ships stats are widely variable based on shipyard level, CSS level, munitions, pilot training etc.

Make the levels more discreet, e.g.
* stats should depend on level of highest shipyard, not current one. The only thing the current shipyard level will alter will be the build speed.
* stats should be more discreet, e.g. only increase for every 5 level changes of the relevant building (shipyard, munitions, css etc.)
* ship name should also remain consistent, e.g. drop the numeric part 'Dory 11' and just keep 'Dory'.

**Comment by lemming552** [comment]
Dory 11 already isn't done as the default name.  Just Dory.
However, people can always change the names, so not sure what the best way to treat ships that are named different should be.   Often that will be the case of a ship from a mission, etc...

**Comment by icydee** [comment]
Yes I saw that after I posted, I was remembering  from an earlier era!

Naming ships differently allows people to create different fleets of ships (even if they have the same spec) but it must be made clear, if they want to combine two fleets they need to rename one fleet to be the same as the other and they will automatically be merged.

**Comment by lemming552** [comment]
We should probably have some good renaming options for ships as well.  Perhaps a rename api that could work off the stats.
